### PR TITLE
[jenkins] Add Mellanox 201811-rpc Jenkins job

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**
Signed-off-by: Andriy Moroz <c_andriym@mellanox.com>

### CODE REVIEW

File: jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+pipeline {
+    agent { node { label 'jenkins-workers-slow' } }
+
```
Comment (lguohan) [inline jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile]: can you add discard and scheduler in your jenkinsfile?

https://github.com/Azure/sonic-build-tools/blob/master/jenkins/mellanox/buildimage-mlnx-201904/Jenkinsfile

Comment (lguohan) [reply jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile]:
```
    options {

        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '30'))



    }



    triggers {

        pollSCM('@midnight')

    }
```

File: jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+'''
+            }
+        }
+
+    }
+    post {
+
+        success {
+            archiveArtifacts(artifacts: 'target/**')
+            cleanWs(disableDeferredWipeout: false, deleteDirs: true, notFailBuild: true)
```
Comment (lguohan) [inline jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile]: this should be in always block.

File: jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile (hunk 1 above)
Comment (andriymoroz-mlnx) [reply jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile]: added

File: jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile (hunk 2 above)
Comment (andriymoroz-mlnx) [reply jenkins/mellanox/buildimage-mlnx-201811-rpc/Jenkinsfile]: done

**Pull request merged**